File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Projects/setattributesfornewtasks.rb
module Asposetasksjava
  module SetAttributesForNewTasks
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'test_tasks.mpp')

        prj = Rjb::import('com.aspose.tasks.Prj')

        # Set new task property
        project.set(prj.NEW_TASK_START_DATE, Rjb::import('com.aspose.tasks.TaskStartDateType').CurrentDate)

        project.save(data_dir + "set_attributes_for_new_tasks.xml", Rjb::import('com.aspose.tasks.SaveFileFormat').XML)

        puts "Set attributes for new tasks, please check the output file."
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithProjects/SetAttributesForNewTasks.php
<?php
namespace Aspose\Tasks\WorkingWithProjects;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\Prj as Prj;
use com\aspose\tasks\TaskStartDateType as TaskStartDateType;
use com\aspose\tasks\SaveFileFormat as SaveFileFormat;

class SetAttributesForNewTasks{

    public static function run($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'test_tasks.mpp');

        $prj = new Prj();

        # Set new task property

        $taskStartDateType=new TaskStartDateType();

        $project->set($prj->NEW_TASK_START_DATE, $taskStartDateType->CurrentDate);

        $saveFileFormat=new SaveFileFormat();
        $project->save($dataDir . "set_attributes_for_new_tasks.xml", $saveFileFormat->XML);
        print "Set attributes for new tasks, please check the output file.".PHP_EOL;
    }
}
?>